homeassistant/components/unifi/diagnostics.py

```diff
@@ -0,0 +1,31 @@
+"""Diagnostics support for UniFi Network."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.system_info import async_get_system_info
+
+from .const import DOMAIN as UNIFI_DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, config_entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    controller = hass.data[UNIFI_DOMAIN][config_entry.entry_id]
+    diag: dict[str, Any] = {}
+
+    diag["home_assistant"] = await async_get_system_info(hass)
```

Comment:
hass system info will be added central with #64670

Reply:
Thanks! I'll remove it